analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CollectionEmptinessChecking.Fixed.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Tests.Diagnostics
+{
+    public class CollectionEmptinessChecking
+    {
+        private static bool HasContent1(IEnumerable<string> l)
+        {
+            return l.Any; // Fixed
```

Comment:
This would be fine for VB, but C# forces devs to be more compiler-friendly 😄 
```suggestion
            return l.Any(); // Fixed
```